src/classes/ui.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Cronbeat;
+
+class UI {
+    public function renderPage($content, $title = 'CronBeat') {
+        $html = <<<HTML
```

Comment:
@jetbrains-junie move all html to a dedicated file, something like "template.html.php". per template also have a "template.view.php" which knows about the needed attributes (and provides the needed setters) and does provide a "render" function that would return the final html code. the template file name should be identical, just the postfix (".html.php" or ".view.php") should be different. all those template files should be in `src/views/`, you can create subfolders there if needed.

views can be nested, so i can have the setup view which internally sets on top of a admin view which sets on top of a base view (just example).